packages/@aws-cdk/aws-lambda-event-sources/lib/sqs.ts

```diff
@@ -80,7 +88,7 @@ export class SqsEventSource implements lambda.IEventSource {
       reportBatchItemFailures: this.props.reportBatchItemFailures,
       enabled: this.props.enabled,
       eventSourceArn: this.queue.queueArn,
-      filters: this.props.filters,
+      filterCriteria: this.props.filterCriteria,
```

Comment:
same here... cannot delete `filters` entirely as that would modify previous behavior.